modular_skyrat/master_files/code/modules/uplink/uplink_items.dm

```diff
@@ -319,6 +317,13 @@
 	cost = 1
 	progression_minimum = 15 MINUTES
 
+/datum/uplink_item/suits/energy_shield_traitor
+	name = "MODsuit Energy Shield Module"
+	desc = "An energy shield module for a MODsuit. The shields can handle up to three impacts \
```

Comment:
Description needs to mention complexity costs

Reply:
Done